src/css/styles/_c-section.scss

```diff
@@ -22,6 +20,29 @@
     margin-top: 1.5rem;
     margin-bottom: 2rem;
     background-color: $col-blue;
+
+    &--right {
+      position: relative;
+      left: calc(100% - 2rem);
+      text-align: end;
+    }
+
+    &--violet {
+      background-color: $col-violet;
+    }
+  }
+
+  &__description {
+    font-size: 1.2rem;
+    line-height: 150%;
+    text-align: justify;
+
+    &--right {
+      position: relative;
+      right: -50%;
+      width: 50%;
+      text-align: end;
+    }
```

Comment:
Why relative?

- [ ] too much relativish

Reply:
Because if postitioned with use of float: right, description is not treated as part of content of the block (description is located below the block- where the margin already starts). I assumed it's neater to include it this way within the scope of the block, than to compensate height of the description (which is dynamic) with extra margins.
Is relative positioning to be avoided when possible?